src/index.ts

```diff
@@ -171,5 +171,7 @@ const printViolationTerminal = (
       // per node
       console.table(nodeViolations)
     }
+  } else {
+    console.log('No accessibility violations detected.')
```

Comment:
Hey, this is a great addition. Would you mind giving it a try with [consola](https://www.npmjs.com/package/consola) to pretty blog the outputs? If you can accommodate the change in this PR, that would be great. Let me know your thoughts.

Reply:
Sure, I cannot see why not. See my latest commits, is that what you had in mind? I was not sure what to do about the `.table` calls.